src/child/components/Favourite.js

```diff
@@ -71,7 +61,15 @@ class Favourite extends Component {
     }
 
     render() {
-        const { stockCode, selected, bindings } = this.props;
+        const { stockCode, selected, bindings, isUnfavouriting } = this.props;
+
+        const { star } = this.refs;
+
+        const modalHeight = 84;
```

Comment:
If these need to be declared here, might be worth sticking them at the top of the file?